docs/content/management-bundles.md

```diff
@@ -112,7 +112,7 @@ bundle has not changed since the last update.
 With the periodic bundle downloading (ie. `short polling`) technique, OPA sends regular requests to the remote HTTP
```

Comment:
Can you remove these changes, please? it's unrelated, looks like some linter made a pass over the file 😅